build.sh

```diff
@@ -200,5 +196,4 @@ else
   install_ocp_tools
   trust_redhat_gpg_keys
   configure_user
-  patch_osbuild
```

Comment:
we are still carrying a patch so we shouldn't drop this?

Reply:
sorry, this is by mistake here , commit-ammend went wrong